bin/main.css

```diff
@@ -1,5 +1,7 @@
+@charset "UTF-8";
```

Comment:
This is the original CSS file, but compiled from the individual SCSS files.
There are no functional changes, just automatic formatting fixes.